apps/chat/src/components/Common/MoveToFolderMobileModal.tsx

```diff
@@ -1,26 +1,38 @@
-import { FloatingOverlay } from '@floating-ui/react';
-import { IconFolderPlus, IconX } from '@tabler/icons-react';
+import { IconFolderPlus } from '@tabler/icons-react';
 import { useCallback, useEffect } from 'react';
 
 import { useTranslation } from 'next-i18next';
 
+import { useScreenState } from '@/src/hooks/useScreenState';
+
+import { ScreenState } from '@/src/types/common';
 import { FolderInterface, MoveToFolderProps } from '@/src/types/folder';
+import { ModalState } from '@/src/types/modal';
 import { Translation } from '@/src/types/translation';
 
+import Modal from './Modal';
+
 interface MoveToFolderMobileModalProps {
   folders: FolderInterface[];
-  onOpen?: () => void;
   onClose: () => void;
   onMoveToFolder: (args: { folderId?: string; isNewFolder?: boolean }) => void;
 }
 
 export const MoveToFolderMobileModal = ({
   folders,
-  onOpen,
   onMoveToFolder,
   onClose,
 }: MoveToFolderMobileModalProps) => {
   const { t } = useTranslation(Translation.SideBar);
+
+  const screenState = useScreenState();
+
+  useEffect(() => {
+    if (screenState !== ScreenState.MOBILE) {
+      onClose();
+    }
+  }, [onClose, screenState]);
```

Comment:
I think this useEffect should be in `ConversationContextMenu.tsx` instead of here
```
  const screenState = useScreenState();

  useEffect(() => {
    if (screenState !== ScreenState.MOBILE) {
      setIsShowExportModal(false);
      setIsShowMoveToModal(false);
    }
  }, [screenState]);
```